logic/atrium-logic-common/src/main/kotlin/ch/tutteli/atrium/logic/creating/basic/contains/creators/impl/ContainsAssertionCreator.kt

```diff
@@ -74,7 +80,19 @@ abstract class ContainsAssertionCreator<T : Any, TT : Any, in SC, C : Contains.C
 
     private fun featureFactory(count: Int, numberOfOccurrences: Translatable): AssertionGroup {
         val assertions = checkers.map { it.createAssertion(count) }
-        return assertionBuilder.feature
+        val checker = checkers.firstOrNull()
+        return if (checkers.size == 1 && checker is AtLeastChecker && checker.times == 1) {
+            assertionBuilder.explanatoryGroup
+                .withDefaultType
+                .withAssertion(
+                    assertionBuilder.explanatory
+                        .withExplanation(descriptionNotFound)
+                        .build()
+                ).let {
+                    if (checker.createAssertion(count).holds()) it
+                    else it.failing
+                }.build()
+        } else assertionBuilder.feature
             .withDescriptionAndRepresentation(numberOfOccurrences, Text(count.toString()))
             .withAssertions(assertions)
             .build()
```

Comment:
Detail but I prefer to have the brackets for both cases if one uses it
```suggestion
        } else {
            assertionBuilder.feature
                .withDescriptionAndRepresentation(numberOfOccurrences, Text(count.toString()))
                .withAssertions(assertions)
                .build()
        }
```